runtime/altair/src/xcm.rs

```diff
@@ -268,13 +269,16 @@ parameter_types! {
 /// Pallet Xcm offers a lot of out-of-the-box functionality and features to
 /// configure and handle XCM messages.
 impl pallet_xcm::Config for Runtime {
+	type AdminOrigin = EnsureRoot<AccountId>;
```

Comment:
**MAYBE CR**

I would opt in here for `EnsureRootOr<AccountId, HalfOfCouncil>`

Reply:
Democracy okay. Leaving that.